libs/TaskManager.js

```diff
@@ -149,6 +153,19 @@ module.exports = class TaskManager{
 			if (task){
 				this.addToRunningQueue(task);
 				task.start(() => {
+					if(task.webhook && task.webhook.length > 3){
```

Comment:
Or in this case, with a value of "ab" (length 2, which passes the test)?

How about "http://invalid-host/invalid-page"? Would the connection error be handled properly?